public/css/styles.css

```diff
@@ -184,6 +184,52 @@ td {
 .address-wrap + button {
   margin-top: 1rem;
 }
+.copy{
```

Comment:
I'd let @darjanin step in to look over css, I'm not too sure about good manners here